Gradata/src/gradata/cloud/_credentials.py

```
@@ -0,0 +1,109 @@
+        log.debug("cloud keyfile read failed: %s", exc)
+        return ""
+
+
+def write_to_keyfile(credential: str) -> Path:
+    """Persist credential to ``~/.gradata/key`` with 0600 permissions."""
+    KEYFILE_DIR.mkdir(parents=True, exist_ok=True)
+    KEYFILE_PATH.write_text(credential.strip() + "\n", encoding="utf-8")
+    try:
+        os.chmod(KEYFILE_PATH, 0o600)
+    except OSError:
+        pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)